annotations/src/main/java/com/statful/client/annotations/Timer.java

```diff
@@ -38,4 +38,9 @@
      * @return An array of aggregations to send to Statful
      */
     Aggregation[] aggregations() default {};
+
+    /**
+     * @return An int of the sampleRate to send to Statful
+     */
+    int sampleRate() default 0;
```

Comment:
I think we're discussing 2 different things here.

First is what (if any) default value should we have for the sample rate when sent through our aspect. After thinking a little better about it I think we should set it to a const `Timer.DEFAULT_SAMPLE_RATE` with a value of `-1` and refactor the aspect implementation to have a private method similar to the following which gets checked after calling the timer method: 

```
isSampled(Timer timer) {
  return Timer.DEFAULT_SAMPLE_RATE != time.sampleRate() && time.sampleRate() > 0;
}
```

This would effectively make the sampleRate optional on the aspect and apply it only if specified, otherwise it'd use the global configured sampleRate on the client.

Second is the ability to enable/disable the sending of the metric for the annotated method. In my mind that's achieved by adding or removing the annotation from the code. @DMB3 maybe you can explain a little better your use-case but it looks to me like that's functionality that's out of the scope of the statful client/aspect?

@reistiago thoughts about this?